src/utils/facet.js

```diff
@@ -21,61 +21,67 @@ export let displayBlankFn = () => <i>Not Specified</i>
 export let Cardinality = _.flow(
   setDisplayName('Cardinality'),
   observer
-)(({ node, tree }) =>
-  _.get('context.cardinality', node) ? (
-    <Flex
-      className="contexture-facet-cardinality"
-      justifyContent="space-between"
-    >
-      <div>
-        Showing{' '}
-        {toNumber(_.min([node.size || 10, _.size(node.context.options)]))} of{' '}
-        {toNumber(node.context.cardinality)}
-      </div>
-      {node.context.cardinality > (node.size || 10) && (
+)(({ node, tree }) => {
+  let size = _.getOr(10, 'size', node)
```

Comment:
This is a slightly different behavior - before, size would be 10 if it was set to 0, but this will change the behavior when you pass in 0 on purpose. Was this change intentional?

Reply:
PAssing size 0 on purpose did not make sense to me... I changed it back.